scripts/devel/create_devel_task_generator.py

```diff
@@ -99,7 +103,26 @@ def main(argv=sys.argv[1:]):
         'build-essential',
         'python3',
     ]
-    if 'catkin' not in pkg_names:
+    if args.build_tool == 'colcon':
```

Comment:
This same list of colcon packages appears in two places and is likely to change over time as a desired colcon extensions grow. Is there somewhere we can centralize this list so that editing need only occur in one place?\
Alternatively I could see cross-commenting to mention that additions may be desired in the other locations.

Another list of colcon packages: https://github.com/ros-infrastructure/ros_buildfarm/pull/585/files#diff-2875658fd26a99ebc3e4697f62421507R528

Reply:
Since the buildfarm doesn't use most of the extensions I removed pretty much all but one in e5af1e520bd22f3ccc2a90acb13491d4c9c0c84b. I doesn't seem to make sense to deduplicate the remaining package name (`colcon-ros`) used in two places.